runtime/executor/src/main/java/org/apache/nemo/runtime/executor/task/SourceVertexDataFetcher.java

```diff
@@ -55,9 +55,9 @@
 
     if (!bounded) {
       this.watermarkTriggerService = Executors.newScheduledThreadPool(1);
-      this.watermarkTriggerService.scheduleAtFixedRate(() -> {
-        watermarkTriggered = true;
-      }, WATERMARK_PERIOD, WATERMARK_PERIOD, TimeUnit.MILLISECONDS);
+      this.watermarkTriggerService.scheduleAtFixedRate(() ->
+        watermarkTriggered = true
+      , WATERMARK_PERIOD, WATERMARK_PERIOD, TimeUnit.MILLISECONDS);
```

Comment:
I think these lines are causing the checkstyle error. Can you append the comma at the end of line 59, instead of at the beginning of line 60?